gecko_client/admin_api.go

```diff
@@ -13,6 +13,10 @@ type AdminApi struct {
 	rpcRequester geckoJsonRpcRequester
 }
 
+type NodeID struct {
+	NodeID string
```

Comment:
Can we add a `json:"nodeid"` tag (or whatever it is) here? I'm trying to be consistent in tagging these so that our variable names don't need to be dependent on the variable names they use in their JSON

Reply:
done